system/Entity.php

```diff
@@ -143,6 +151,28 @@ public function fill(array $data = null)
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Set boolean value for entity state.
+	 *
+	 * @param boolean $exists
+	 */
+	public function setExists(bool $exists)
```

Comment:
Yes, because the exist State is controlled outside of the Model object context for example by the model save model if its an Database related entity.